### ISSUE TITLE
Navbar Links should have higher contrast

### ISSUE THREAD

**Issue by ashleywxwx** [AndrewBell/menteaship-home#3]
Navbar links should be updated to have a higher contrast ratio.

As a user
When viewing the navbar
Then the links should have high contrast, so they are easy to read
